doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -163,31 +173,22 @@ If you experience any problems during the process, restart the Programmer app by
 
       Programmer - Add file
 
-#. Navigate to where you extracted the firmware, and choose the :file:`mfwnrf9160_<version-number>.zip` file.
-#. Click :guilabel:`Write` in the :guilabel:`Device` section of the side panel.
-
-   .. figure:: /images/programmer_write_nrf9160dk.png
-      :alt: Programmer - Write
-
-      Programmer - Write
-
-   The :guilabel:`Modem DFU` dialog window appears.
-
-   .. figure:: images/programmerapp_modemdfu.png
-      :alt: Modem DFU window
-
-      The Modem DFU window
-
-#. Click the :guilabel:`Write` button in the :guilabel:`Modem DFU` dialog window to update the firmware.
-   Do not unplug or turn off the device during this process.
+#. Navigate to where you extracted the firmware, and then to the :file:`img_app_bl` folder there.
+#. Select either :file:`nrf9160dk_asset_tracker_v2_ltem_<version-number>.hex` (LTE-M mode) or :file:`nrf9160dk_asset_tracker_v2_nbiot_<version-number>.hex` (NB-IoT mode), depending on where you are located.
+   Check with your SIM card provider for the mode supported at your location.
+   If you are using the iBasis SIM card provided with the DK, you can see `iBasis IoT network coverage`_ .
+   You can use either mode if your location has support for both.
 
-When the update is complete, you see a success message.
+   For NB-IoT, there is a second variant of the firmware in the :file:`nrf9160dk_asset_tracker_v2_nbiot_legacy_pco_<version-number>.hex` file.
+   Only use this legacy variant if your network does not support ePCO.
 
-.. note::
+#. Click the :guilabel:`Erase & write` button in the :guilabel:`Device` section to program the DK.
+   Do not unplug or turn off the DK during this process.
 
-   If you experience problems updating the modem firmware, click :guilabel:`Erase all` in the :guilabel:`Device` section of the side panel and try updating again.
+   .. figure:: /images/programmer_erasewrite_nrf9160dk.png
+      :alt: Programmer - Erase & write
 
-   Doing this erases the contents of the flash memory, so the application must be programmed again to the DK.
+      Programmer - Erase & write
```

Comment:
Same comment as line 121. Might be nice to check this for all the added screenshots.